Caprica/pex/PexFunctionBuilder.cpp

```diff
@@ -40,6 +40,10 @@ void PexFunctionBuilder::populateFunction(PexFunction* func, PexDebugFunctionInf
     line = reportingContext.getLocationLine(l, line);
     if (line > std::numeric_limits<uint16_t>::max())
       reportingContext.fatal(l, "The file has too many lines for the debug info to be able to map correctly!");
+    // check that the line is larger than the previous one; if not, then set it to the previous one.
+    // Compiler does not like
```

Comment:
Would it be reasonable to hide this behind an engine limit?

Also, this comment is incomplete.